src/main/java/org/kohsuke/github/Requester.java

```diff
@@ -576,11 +576,20 @@ private InputStream wrapStream(InputStream in) throws IOException {
         InputStream es = wrapStream(uc.getErrorStream());
         try {
             if (es!=null) {
+                String errorString = IOUtils.toString(es, "UTF-8");
+                // Check to see whether we hit a 403, and the message indicates the
+                // abuse handler (occurs on too many concurrent requests)
+                if (responseCode == HttpURLConnection.HTTP_FORBIDDEN && 
+                    errorString.contains("You have triggered an abuse detection mechanism.")) {
```

Comment:
@kohsuke This is the only bit I'm not totally sure about.  The best bit would be to just be able to check for 403, but I'm not sure whether 403 happens in more circumstances than just API abuse.  There wasn't anything else in the header indicating abuse (though maybe I could just check for Retry-After)

Additionally, I imagine that there isn't a way to get a localized error string here, but if there was the check would be broken.  Thoughts?